src/test/java/com/wikia/webdriver/common/core/imageutilities/SzogiScreenshotTests.java

```diff
@@ -0,0 +1,69 @@
+package com.wikia.webdriver.common.core.imageutilities;
+
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.api.ArticleContent;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.article.ArticlePageObject;
+import org.testng.annotations.Test;
+
+
+    @Test(groups = "szogi_dcreenshot_tests")
+    @Execute(onWikia = "wikiActivities", asUser = User.WIKIACTIVITY_USER)
+    public class SzogiScreenshotTests extends NewTestTemplate {
+
+        private User testUser = User.WIKIACTIVITY_USER;
+
+        @Test
+        public void webElementInViewport() {
+            ArticlePageObject article = createArticle();
+            article.getScreenshotOfArticleName();
+
+        }
+
+        @Test
+        public void webElementBelowViewport() {
+            ArticlePageObject article = createArticle();
+            article.getScreenshotOfSubmitCommentButton();
+
+        }
+
+        @Test
+        public void webElementOverViewport() {
+            ArticlePageObject article = createArticle();
+            article.getScreenshotOfArticleNameAfterScroll();
+        }
+
+        //Firefox needs dpr=1 for mac?!?!?!?!?
+
+//        @InBrowser(browser = Browser.FIREFOX)
+//        @Test
+//        public void webElementInViewportInFirefox() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)